components/zerotier/zerotier.app.mjs

```diff
@@ -32,6 +32,48 @@ export default {
         }));
       },
     },
+    memberHidden: {
+      type: "boolean",
+      label: "Member Hidden?",
+      description: "Whether or not the member is hidden in the UI",
+      optional: true,
+      default: null,
```

Comment:
Since `memberHidden` is boolean, the default should be either `true` or `false`. Another option would be to not include any default value. Then if the user doesn't enter anything, the value would be undefined.
```suggestion
      default: false,
```